_application_development/github/protected-branches.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Protected branches
+position: 3
+---
+
+There's only one rule about branches: **anything in the `master` branch is always deployable**.
```

Comment:
There's only one rule about the master branch.

Maybe explaining the flow of:
Branch from master
Do work
PR
Merge

should happen in a separate file, and this should focus on why master is protected